packages/trpc-server/src/index.ts

```diff
@@ -44,7 +44,7 @@ export const trpcServer = ({
               return Reflect.get(t, p, t)
             },
           }),
-    }).then((res) => c.body(res.body, res))
+    }).then((res) => c.newResponse(res.body, res))
```

Comment:
The types for `c.body` don't allow the combination of..

```ts
(data: Data | null, init?: ResponseOrInit): Response;
```

..but `c.newResponse` does, both end up calling `#newResponse`

https://github.com/honojs/hono/blob/0bb821a4a10053f82a6afac09a1243fd2da62ad4/src/context.ts#L730-L738

https://github.com/honojs/hono/blob/0bb821a4a10053f82a6afac09a1243fd2da62ad4/src/context.ts#L707

Reply:
Both `c.body` and `c.newResponse` are okay, but `c.body` is better since `c.newResponse` is not often used as a public API. So how about adding `@ts-expected-error` like this (I think it's not a big problem adding `@ts-expect-error` or `@ts-ignore` internal implementation)?

```ts
 // @ts-expect-error c.body accepts both ReadableStream and null but is not typed well
```

Reply:
Great! Done 👍🏻